lib/solidus_bactracs/api/request_runner.rb

```diff
@@ -117,7 +117,7 @@ def rma_exists?(response)
       end
 
       def shipment_synced(shipment)
-        shipment.update_column(:bactracs_synced_at, Time.zone.now)
+        shipment.update_attribute(:bactracs_synced_at, Time.zone.now)
```

Comment:
Why is this needed?

It will save other modifications as well as update the referenced field